styles.css

```diff
@@ -0,0 +1,34 @@
+* {
+  font-family: "Raleway";
+}
+
+body {
+  background-image: url(https://images.unsplash.com/photo-1526289034009-0240ddb68ce3?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=871&q=80);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-size: cover;
+}
+
+#title {
+  font-weight: bold;
+  margin-top: 50px;
+  margin-bottom: 10px;
+}
+
```

Comment:
- [ ] use an indentation of 2 spaces. Please fix it

Reply:
- [ ] It is a good practice to indent your code with 2 spaces. Since it improves readability. Why don't you add indentation of 2 spaces?